app/pages/nonsipp/sharesdisposal/CompanyNameOfSharesBuyerPage.scala

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package pages.nonsipp.sharesdisposal
+
+import play.api.libs.json.JsPath
+import models.SchemeId.Srn
+import models.Money
+import pages.QuestionPage
+import config.Refined._
+import utils.RefinedUtils._
+import eu.timepit.refined.refineMV
+
+case class CompanyNameOfSharesBuyerPage(srn: Srn, index: Max5000, secondaryIndex: Max50) extends QuestionPage[String] {
```

Comment:
Sorry @westwater, based on conversations we had at the sprint start I assumed it will be Christo and Ben working on this journey, so we took a bit different approach here and already added pages for this section (share seller's details). This is the page for this: https://github.com/hmrc/pension-scheme-return-frontend/blob/01d30ec7c7512c79d90fd28f8b49447ce0d06069/app/pages/nonsipp/sharesdisposal/CompanyBuyerNamePage.scala#L25 and these are all the other pages added in the journey: https://github.com/hmrc/pension-scheme-return-frontend/blob/01d30ec7c7512c79d90fd28f8b49447ce0d06069/app/pages/nonsipp/sharesdisposal/WhoWereTheSharesSoldToPage.scala#L38 Sorry for the confusion, let me know please if you'd like to discuss.

Reply:
No worries 👍 removed the page